test/restore.bats

```diff
@@ -12,90 +12,51 @@ function teardown() {
 
 @test "crio restore" {
 	start_crio
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-
-	run crictl pods --quiet --id "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_list_info="$output"
-
-	run crictl inspectp -o json "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	pod_list_info=$(crictl pods --quiet --id "$pod_id")
+
+	output=$(crictl inspectp -o json "$pod_id")
 	pod_status_info=$(echo "$output" | jq ".status.state")
```

Comment:
Since `jq` ignores empty input (see previous comment), there should be a check for non-empty output added here. Can be addressed either right here or separately.